_layouts/page.html

```diff
@@ -1,20 +1,19 @@
----
-layout: default
----
-{% assign author = site.authors[page.author] %}
-<div id="content" class="site" role="main">
-  <article role="article" itemscope itemtype=http://schema.org/BlogPosting>
-    <meta itemprop="description" content="{{ page.excerpt | strip_html}}">
-    <header>
-      <h3 itemprop="name">{{ page.title }}</h3>
-    </header>
-    <section itemprop="articleBody">
-      {{ content }}
-    </section>
-    <footer role="contentinfo">
-      {% include comments.html %}
-    </footer>
-    <meta itemprop="url" content="{{ site.url }}{{ page.url }}">
-  </article>
+{% include head.html %}
+
+<div class="pageheader">
```

Comment:
i'm only pointing out this naming convention as it should be `page-header` because we use `page-section` and other class naming with the `-`